model/project_ref.go

```diff
@@ -117,6 +117,8 @@ type ProjectRef struct {
 	ContainerSizeDefinitions []ContainerResources `bson:"container_size_definitions,omitempty" json:"container_size_definitions,omitempty" yaml:"container_size_definitions,omitempty"`
 	ContainerSecrets         []ContainerSecret    `bson:"container_secrets,omitempty" json:"container_secrets,omitempty" yaml:"container_secrets,omitempty"`
 
+	// RepoRefId is the project ref id that this project ref tracks, if any (i.e. this is a branch
```

Comment:
This field is actually the ID of the repo ref (which is a separate document stored in a [separate collection](https://gist.github.com/hadjri/ffd4ad5aeb0ddc6d51d99a160dd04745#file-repo_ref-go-L18)) that a project is attached to, so we shouldn't refer to it as a project ref.

I think we can remove the parentheses statement, the first statement is succinct enough.